src/components/ArticleById.jsx

```diff
@@ -19,19 +20,24 @@ function ArticleById() {
   }
   useEffect(fetchSingleArticle, []);
   return (
-    <div className="single-article-card">
-      <h2 id="single-article-title">{singleArticle.title}</h2>
-      <div className="single-article-author">
-        <p>From {singleArticle.author}</p>
+    <>
+      <div className="single-article-card">
+        <h2 id="single-article-title">{singleArticle.title}</h2>
+        <div className="single-article-author">
+          <p>From {singleArticle.author}</p>
+          <p id="single-article-votes">Votes: {singleArticle.votes}</p>
+        </div>
+        <div className="single-article-image">
+          <img src={singleArticle.article_img_url} />
+        </div>
+        <div className="single-article-body">
+          <p>{singleArticle.body}</p>
+        </div>
+        <p>Comments: {singleArticle.comment_count}</p>
+        <button onClick={handleBackClick}>Back to Articles</button>
```

Comment:
Lots of Divs and p tags beings used. Other HTML tags are available :) Try keep them as semantic as possible